apps/SageAccountWeb/src/components/ResetPassword.tsx

```diff
@@ -159,15 +150,12 @@ export const ResetPassword = (props: ResetPasswordProps) => {
               </form>
             </>
           ) : (
-            <Box sx={{ position: 'relative' }}>
-              <IconButton
```

Comment:
Position relative to LeftPanel so that the back button can use the same styling as in other locations